agent/command/exec.go

```diff
@@ -348,14 +348,7 @@ func (c *subprocessExec) runCommand(ctx context.Context, taskID string, cmd *jas
 		}
 	}
 
-	if c.ContinueOnError {
-		logger.Execution().Notice(message.WrapError(err, message.Fields{
-			"task":       taskID,
-			"binary":     c.Binary,
-			"background": c.Background,
-			"silent":     c.Silent,
-			"continue":   c.ContinueOnError,
-		}))
+	if c.ContinueOnError && err != nil {
 		logger.Execution().Noticef("Script errored, but continue on error is set - continuing task execution. Error: %s.", err)
```

Comment:
Do we no longer want the message.Fields here?

Reply:
I'm thinking it's kind of redundant, and we don't have it at the other place we do this. I'd prefer to remove it.